dataflow-deploy/README.md

```diff
@@ -0,0 +1,75 @@
+# dataflow-deploy
+
+This is a GitHub Action to deploy dataflow jobs.
+
+## Usage
+
+See [action.yml](action.yml).
+
+### Examples
+
+```yaml
+jobs:
+  deploy-dataflow-job:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: extenda/actions/gcp-secret-manager@v0
+        with:
+          service-account-key: ${{ secrets.SECRET_AUTH }}
+          secrets: |
+            DATAFLOW_SERVICE_ACCOUNT: dataflow-worker
+
+      - name: Determine version
+        uses: extenda/actions/conventional-version@v0
+        id: semver
+        with:
+          build-number: ${{ github.run_number }}
+
+      - name: Dataflow deploy
+        uses: extenda/actions/dataflow-deploy@v0
+        with:
+          service-account-key: ${{ secrets.GCLOUD_AUTH_STAGING }} # Used to authenticate with gcloud
+          dataflow-service-account: ${{ env.DATAFLOW_SERVICE_ACCOUNT }}
+          job-type: job
+          job-name: send-to-bigquery
+          job-version: ${{ github.run_number }} # Used for job name seperation (will be appended to job name)
```

Comment:
Is there any reason to use run number vs. something that's easier to track like the short SHA?